neo-gui/UI/TransferDialog.Designer.cs

```diff
@@ -36,6 +36,8 @@ private void InitializeComponent()
             this.button3 = new System.Windows.Forms.Button();
             this.button2 = new System.Windows.Forms.Button();
             this.groupBox1 = new System.Windows.Forms.GroupBox();
+            this.comboBox2 = new System.Windows.Forms.ComboBox();
+            this.label3 = new System.Windows.Forms.Label();
```

Comment:
Maybe next time we could use more clear names for these components. It is hard to review the code with these names.

Reply:
@lock9 It have been tested about UTXO and NEP5 asset by NGD test OP. Function works all right. 

About name, I agree we should take better name. In next version, I will try to do that.